controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java

```diff
@@ -261,7 +264,7 @@ public InternalStepRunner(Controller controller) {
                     // Same as CERTIFICATE_NOT_READY above, only from the controller
                     logger.log("Waiting for certificate to become valid: New application, or old one has expired");
                     if (startTime.plus(timeouts.endpointCertificate()).isBefore(controller.clock().instant())) {
-                        logger.log("Controller could not validate certificate within " +
+                        logger.log(WARNING, "Controller could not validate certificate within " +
```

Comment:
A few more of these log messages should probably have their level raised in the application deployment log, but unfortunately this level also applies to our log

Reply:
Can split the logging to deployment/vespa-log. It wasn't such a great idea to keep them together anyway ;)